### PR TITLE
YLP-2292 carbs and bolus average on trends

### PR DESCRIPTION
**Pull request by fatb38**
## Checklist
- [x] PR name contains the Jira ticket number
- [x] Changelog updated
- [x] Code tested (unit tests and integration tests)
- [x] No console errors/warnings in integration tests
- [x] No translation files updated (except for English files)
- [x] Code formatted on every file
- [x] Copyright added on every file
- [x] End-to-end tests successful on CI

### CODE REVIEW

File: packages/yourloops/components/carbs-and-bolus/carbs-and-bolus-cell.tsx [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+  return (
+    <Box
+      width="12.5%"
+      sx={{ ...sx, borderLeft: LIGHT_BORDER }}
+      data-testid="carbs-and-bolus-cell"
+    >
+      <Typography
+        variant="caption"
+        sx={{ marginLeft: '4px' }}
```
Comment (TimCanu) [inline packages/yourloops/components/carbs-and-bolus/carbs-and-bolus-cell.tsx:72]: Same question for the theme.spacing thingy?

File: packages/yourloops/components/carbs-and-bolus/carbs-and-bolus-cell.tsx [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+      <Box className={classes.cellsWrapper}>
+        <Box
+          className={classes.cell}
+          sx={{ backgroundColor: carbsCellBackgroundColor }}
+          onMouseEnter={() => openPopover(TypeOfCell.RescueCarbs)}
+          onMouseLeave={closePopover}
+        >
+          <Typography
+            variant="caption"
+            sx={{ color: '#444444' }}
```
Comment (TimCanu) [inline packages/yourloops/components/carbs-and-bolus/carbs-and-bolus-cell.tsx:85]: **❔ question:** ‏Aren't we supposed to use color from mui or defined global variables for colors?

File: packages/yourloops/components/carbs-and-bolus/carbs-and-bolus-cell.tsx [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+
+        <Box
+          className={classes.cell}
+          sx={{ backgroundColor: bolusCellBackgroundColor }}
+          onMouseEnter={() => openPopover(TypeOfCell.ManualBolus)}
+          onMouseLeave={closePopover}
+        >
+          <Typography
+            variant="caption"
+            sx={{ color: 'white' }}
```
Comment (TimCanu) [inline packages/yourloops/components/carbs-and-bolus/carbs-and-bolus-cell.tsx:103]: Same question? theme.palette.common.white

**Review by TimCanu — APPROVED** [review]
You have sonar issues to fix

File: locales/en/yourloops.json [hunk 4]
```diff
@@ -16,8 +16,8 @@
   "accept-notification-success": "You have accepted the invitation from {{name}}",
   "accompanying-documents": "Accompanying documents",
   "account-created-info-1": "Congratulations, you have just finished creating your new YourLoops account",
-  "account-created-info-2-hcp": "Clicking the continue button will take you to your dashboard. You can then add your patients, and also join care teams",
   "account-created-info-2-caregiver": "Clicking the continue button will take you to your dashboard. Patients must invite you from their YourLoops account for you to access their data.",
+  "account-created-info-2-hcp": "Clicking the continue button will take you to your dashboard. You can then add your patients, and also join care teams",
```
[the remark this answers is not in the document]
Comment (fatb38) [reply locales/en/yourloops.json:20]: keyboard/chair problem ?

**Pull request merged**